experiment/molecule/src/common.ts

```diff
@@ -14,55 +15,303 @@ export const Uint8: FixedBinaryCodec<number> = {
     return new DataView(buf).getUint8(0);
   },
 };
-export declare const HexUint8: FixedBinaryCodec<string>;
+export const HexUint8: FixedBinaryCodec<string> = {
+  __isFixedCodec__: true,
+  byteLength: 1,
+  pack(str) {
+    return Uint8.pack(parseInt(str, 16));
```

Comment:
we should check if `str` is valid HexString here

Reply:
done